src/models/IKilpiVia.ts

```diff
@@ -0,0 +1,11 @@
+interface IKilpiViaPrimaryKey {
+    relid: number;
+}
+
+export default interface IKilpiVia extends IKilpiViaPrimaryKey {
+    viaruotsi: string,
+    viasuomi: string
```

Comment:
These fields should be named in english like in other models:

```
interface IKilpiViaPrimaryKey {
    id: number;
}
export default interface IKilpiVia extends IKilpiViaPrimaryKey {
    nameSw: string,
    nameFi: string
}
```
Then create
```
IExternalKilpiVia {
    relid: string,
    viaruotsi: string,
    viasuomi: string
}
```

However, since this IKilpiVia is only used at routePathView, i would define interface IExternalKilpiVia inside kilpiViaService and "hide it there". Other components doenst have to know about it. Just map IExternalKilpiVia into IKilpiVia and let that kilpiViaService return it